src/components/Student.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+import { Heading, IconButton } from '@chakra-ui/react'
+import { DeleteIcon } from '@chakra-ui/icons'
+import { useFirestore, useUser } from 'reactfire'
+
+type StudentProps = {
+  studentName: string
+  studentInStudentGroupId: string
+}
+
+const StudentContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 1px solid black;
+  padding: 10px;
+`
```

Comment:
consider using the `Flex` Chakra-UI component here. We're incurring the expense of a component library after all, in for a penny in for a pound.

Reply:
Okay, I switched it out for the Flex component.  I didn't combine it with emotion though.  Let me know what you think.